marketplace/wpp_templates/requests.py

```diff
@@ -0,0 +1,33 @@
+import requests
+
+from marketplace.core.types.channels.whatsapp_base.exceptions import FacebookApiException
+
+
+class TemplateMessageRequest(object):
+    def __init__(self, access_token: str) -> None:
+        self._access_token = access_token
+
+    @property
+    def _headers(self) -> dict:
+        return {"Authorization": f"Bearer {self._access_token}"}
+
+    def create_template_message(self, waba_id: str, name: str, category: str, components: list, language: str) -> dict:
+        params = dict(
+            name=name, category=category, components=components, language=language, access_token=self._access_token
+        )
+
+        response = requests.post(url=f"https://graph.facebook.com/14.0/{waba_id}/message_templates", params=params)
```

Comment:
There is an environment variable that already has this url mounted: https://github.com/weni-ai/weni-integrations-engine/blob/main/marketplace/settings.py#L280
Line:
- [28](https://github.com/weni-ai/weni-integrations-engine/pull/167/files#diff-d643a263a1979362e1dd535b2a0ccb70d9c90021135858806b457615afa866ecR28)